package.json

```diff
@@ -66,7 +66,8 @@
     "eslint-plugin-ember": "^5.2.0",
     "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.8.0"
+    "qunit-dom": "^0.8.0",
+    "yapp-test-support": "git+ssh://git@github.com/yappbox/yapp-test-support.git#v2.2.0"
```

Comment:
This is a problem, right? Test-support is private. Maybe we can publish it as a scoped package?